### PR TITLE
Correct notebooks for neural networks in docs 

### PR DESCRIPTION
**Pull request by Baschdl**
Loss was calculated using `np.sum` without accounting for the number of samples. This worked for the batch size of `128` in combination with the low learning rate but lead to exploding gradients for larger batches.
`np.mean` is used now with a larger learning rate to achieve similar results.

### CODE REVIEW

**Comment by mattjj** [comment]
Thanks for the fix!
